src/ActorUtil.cpp

```diff
@@ -19,10 +19,13 @@
 
 #include "arch/Dialog/Dialog.h"
 
+#include <unordered_map>
+
+using std::string;
 using std::vector;
 
 // Actor registration
-static std::map<RString,CreateActorFn> *g_pmapRegistrees = nullptr;
+static std::unordered_map<string,CreateActorFn> *g_pmapRegistrees = nullptr;
```

Comment:
Instead of worrying about this being null, wouldn't it make more sense to just have this be a `std::unique_ptr` and be done with it?